.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     name: 👷 ${{ matrix.cloud }}
     strategy:
       matrix:
-        cloud: [aws, aws_minimal, azure, gcp]
+        cloud: [aws, azure, gcp]
```

Comment:
You definitely meaning to remove the `aws_minimal` item?

Reply:
Yeah @b4k3r accidentally forgot it in [this pr](https://github.com/spacelift-io/spacelift-worker-image/pull/50)

Reply:
Yeah, I completely forgot about it. Thanks @peterdeme .